api/compliance.go

```diff
@@ -26,6 +26,10 @@ type ComplianceService struct {
 	client *Client
 }
 
+type CloudComplianceReport interface {
+	GetComplianceRecommendations() []ComplianceRecommendation
+}
+
```

Comment:
🤔 - It looks easy to do this kind of things but, this looks like a very CLI specific use case. Not sure about adding it.